test/test_pipeline.py

```diff
@@ -1072,6 +1074,20 @@ def test_split_document_list_content_type(test_docs_xs):
     assert result["output_3"][0].meta["meta_field"] == "test5"
 
 
+@pytest.mark.parametrize("join_mode", ["concatenate", "merge"])
+def test_join_answers_concatenate(join_mode):
```

Comment:
`test_join_answers_concatenate` is a little bit misleading as you test for "concatenate" and "merge".